2018-10-24 19:58:19,177 INFO: Microblog startup [in /Users/katie/Documents/CS205/lgbt_music/app/__init__.py:46]
2018-10-24 20:01:38,000 INFO: Microblog startup [in /Users/katie/Documents/CS205/lgbt_music/app/__init__.py:46]
2018-10-24 20:01:45,809 ERROR: Exception on /new_venue [GET] [in /Users/katie/local_music/lib/python3.7/site-packages/flask/app.py:1761]
Traceback (most recent call last):
  File "/Users/katie/local_music/lib/python3.7/site-packages/flask/app.py", line 2292, in wsgi_app
    response = self.full_dispatch_request()
  File "/Users/katie/local_music/lib/python3.7/site-packages/flask/app.py", line 1815, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "/Users/katie/local_music/lib/python3.7/site-packages/flask/app.py", line 1718, in handle_user_exception
    reraise(exc_type, exc_value, tb)
  File "/Users/katie/local_music/lib/python3.7/site-packages/flask/_compat.py", line 35, in reraise
    raise value
  File "/Users/katie/local_music/lib/python3.7/site-packages/flask/app.py", line 1813, in full_dispatch_request
    rv = self.dispatch_request()
  File "/Users/katie/local_music/lib/python3.7/site-packages/flask/app.py", line 1799, in dispatch_request
    return self.view_functions[rule.endpoint](**req.view_args)
  File "/Users/katie/local_music/lib/python3.7/site-packages/flask_login/utils.py", line 261, in decorated_view
    return func(*args, **kwargs)
  File "/Users/katie/Documents/CS205/lgbt_music/app/routes.py", line 76, in new_venue
    if form.validate_on_submit():
NameError: name 'form' is not defined
2018-10-24 20:01:59,522 INFO: Microblog startup [in /Users/katie/Documents/CS205/lgbt_music/app/__init__.py:46]
